### PR TITLE
fix(storage): fix  failed micro-bench

### PR DESCRIPTION
**Pull request by Little-Wallace**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## Checklist

- [x] I have written necessary rustdoc comments
- [ ] I have added necessary unit tests and integration tests
- [ ] I have added test labels as necessary. See [details](https://github.com/risingwavelabs/risingwave/blob/main/docs/developer-guide.md#ci-labels-guide).
- [ ] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [ ] My PR contains breaking changes. (If it deprecates some features, please create a tracking issue to remove them in the future).
- [ ] All checks passed in `./risedev check` (or alias, `./risedev c`)
- [ ] My PR changes performance-critical code. (Please run macro/micro-benchmarks and show the results.)

- [ ] My PR contains critical fixes that are necessary to be merged into the latest release. (Please check out the [details](https://github.com/risingwavelabs/risingwave/blob/main/CONTRIBUTING.md))

## Documentation

- [ ] My PR needs documentation updates. (Please use the **Release note** section below to summarize the impact on users)

## Release note

If this PR includes changes that directly affect users or other significant modifications relevant to the community, kindly draft a release note to provide a concise summary of these changes. Please prioritize highlighting the impact these changes will have on users.

</details>

### CODE REVIEW

File: src/storage/benches/bench_compactor.rs [hunk 1]
```diff
@@ -220,15 +220,15 @@ fn bench_merge_iterator_compactor(c: &mut Criterion) {
     let sstable_store = mock_sstable_store();
     let test_key_size = 256 * 1024;
     let info1 = runtime
-        .block_on(async { build_table(sstable_store.clone(), 1, 0..test_key_size, 1).await });
+        .block_on(async { build_table(sstable_store.clone(), 1, 0..test_key_size/2, 1).await });
```
Comment (kwannoel) [inline src/storage/benches/bench_compactor.rs:223]: Why divide key size by 2?

File: src/storage/benches/bench_compactor.rs [hunk 2]
```diff
@@ -175,7 +175,7 @@ fn bench_table_scan(c: &mut Criterion) {
 
     c.bench_function("bench_table_iterator", |b| {
         let info1 = info.clone();
-        b.to_async(FuturesExecutor)
+        b.to_async(&runtime)
```
Comment (kwannoel) [inline src/storage/benches/bench_compactor.rs:178]: I guess this is the main source of the bug?

**Review by kwannoel — APPROVED** [review]

File: src/storage/benches/bench_compactor.rs (hunk 2 above)
Comment (Little-Wallace) [reply src/storage/benches/bench_compactor.rs:178]: maybe

File: src/storage/benches/bench_compactor.rs (hunk 1 above)
Comment (Little-Wallace) [reply src/storage/benches/bench_compactor.rs:223]: Because it would cause panic because of duplicated key

**Pull request merged**